fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2735,6 +2737,81 @@ private List<ResultRow> convertResultBatchToResultRows(TResultBatch batch) {
         return resultRows;
     }
 
+    public void generateStreamLoadNereidsPlan(TUniqueId queryId) throws Exception {
+        LOG.info("TUniqueId: {} generate stream load plan", queryId);
+        context.setQueryId(queryId);
+        context.setStartTime();
```

Comment:
context.setStartTime();
profile.getSummaryProfile().setQueryBeginTime();
Is these useful?